crates/flowctl/src/generate/mod.rs

```diff
@@ -288,6 +288,14 @@ async fn generate_missing_materialization_configs(
             },
             serde_json::from_str::<url::Url>(config.config.get()).ok(),
         ),
+        models::MaterializationEndpoint::Dekaf(config) => (
+            materialize::request::Spec {
+                connector_type: flow::materialization_spec::ConnectorType::Dekaf as i32,
+                config_json: serde_json::to_string(config).unwrap(),
+            },
+            // Dekaf isn't a pluggable connector, and so does not have dynamic config.
```

Comment:
nit: It does have a config, though -- it's just empty today (and probably we'll add to it in the future as we allow more customization of behavior). Ideally we'd be able to generate stubs here for it based on the JSON schema of `DekafConfig`, just as we generate stubs for the dynamic endpoint config schemas of connectors. I don't recall this code well enough to know how much of a lift that is.

Reply:
Updated to support either a string (in which case it's assumed to be a file or url and gets loaded if valid), or an object in which case it's an inlined `DekafConfig`